data/data-pipeline/data_pipeline/score/field_names.py

```diff
@@ -196,3 +197,58 @@
+AGRICULTURE_LOSS_RATE_LOW_INCOME = f"At or above the {PERCENTILE}th percentile for agriculture and is low income"
+EXPECTED_BUILDING_LOSS_RATE_LOW_INCOME = f"At or above the {PERCENTILE}th percentile for building loss and is low income"
+
+# Clean energy and efficiency
+PM25_LOW_INCOME = "PM2.5 exposure and is low income"
+ABOVE_90TH_FOR_COST_BURDEN_LOW_INCOME = f"At or above the {PERCENTILE}th percentile for energy cost burden score and is low income"
+# Clean transportation
+DIESEL_PARTICULATE_MATTER_LOW_INCOME = f"At or above the {PERCENTILE}th percentile for diesel particulate matter and is low income"
+TRAFFIC_PROXIMITY_MATTER_LOW_INCOME = "Traffic Proximity and is low income"
+# Affordable and Sustainable Housing
+LEAD_PAINT_HOME_VALUE = f"At or above the {PERCENTILE}th percentile for lead paint and the median house value is less than 90th percentile"
+HOUSING_BURDEN_LOW_INCOME = (
+    f"At or above the {PERCENTILE}th percentile for housing cost burden and is low income"
+)
+# Remediation and Reduction of Legacy Pollution
+RMP_LOW_INCOME = f"At or above the {PERCENTILE}th percentile for proximity to RMP sites and is low income"
+SUPERFUND_LOW_INCOME = f"At or above {PERCENTILE}th percentile for proximity to superfund sites and is low income"
+HAZARDOUS_WASTE_LOW_INCOME = f"At or above above {PERCENTILE}th percentile for proximity to hazardous waste facilities and is low income"
+# Critical Clean Water and Waste Infrastructure
+WASTEWATER_LOW_INCOME = f"At or above above 90th percentile for wastewater discharge and is low income"
+# Health Burden
+DIABETES_LOW_INCOME = (
+    f"At or above the {PERCENTILE}th percentile for diabetes and is low income"
+)
+ASTHMA_LOW_INCOME = (
+    f"At or above the {PERCENTILE}th percentile for asthma and is low income"
+)
+HEART_DISEASE_LOW_INCOME = (
+    f"At or above the {PERCENTILE}th percentile for heart disease and is low income"
+)
+LIFE_EXPECTANCY_INCOME = (
+    f"At or above the {PERCENTILE}th percentile for life expectancy and is low income"
+)
+
+ACHIEVEMENT_TEMPLATE_FOR_WORKFORCE = """and where the high school degree achievement 
+rate for adults 25 years and older is less than """
+
+# Workforce
+UNEMPLOYMENT_LOW_HS_EDUCATION = f"""At or above the {PERCENTILE}th percentile for unemployment" 
+   {ACHIEVEMENT_TEMPLATE_FOR_WORKFORCE} {PERCENTILE}%"""
+
+LINGUISTIC_ISOLATION_LOW_HS_EDUCATION = f"""At or above the {PERCENTILE}th percentile
+    for linguistic isolation {ACHIEVEMENT_TEMPLATE_FOR_WORKFORCE} {PERCENTILE}%"""
+
+POVERTY_LOW_HS_EDUCATION  = f"""At or above the {PERCENTILE}th percentile for households
+    at or below 100 percent of federal poverty level {ACHIEVEMENT_TEMPLATE_FOR_WORKFORCE} {PERCENTILE}%"""
+
+MEDIAN_INCOME_LOW_HS_EDUCATION = f"""At or below the {PERCENTILE}th percentile for median
+    income {ACHIEVEMENT_TEMPLATE_FOR_WORKFORCE} {PERCENTILE}%"""
+
+THRESHOLD_COUNT = "Total Number of Factor Criterion Met"
```

Comment:
This is a real nitpick, but maybe something like "Total threshold criteria exceeded". Thanks!

Reply:
Got it